plugins/chain_plugin/include/eosio/chain_plugin/blockvault_sync_strategy.hpp

```diff
@@ -76,7 +81,7 @@ struct blockvault_sync_strategy : public sync_callback {
 
       try {
          ++_num_blocks_received;
-         _blockchain_provider.incoming_block_sync_method(block, block->calculate_id());
+         _blockchain_provider.incoming_blockvault_sync_method(block);
```

Comment:
Should EOS_ASSERT that this returned true.

Reply:
added assertion